apis/classes/Event.js

```diff
@@ -0,0 +1,9 @@
+function Event(name, startTime, endTime, location, summary) {
```

Comment:
Because this is a function, you can call `Event()` instead of `new Event()` and it will return undefined without throwing an error – let's turn this into an actual class.